lsp-clients.el

```diff
@@ -235,5 +235,32 @@ PARAMS progress report notification data."
                   :server-id 'php-ls))
 
 
+;; C-family (C, C++, Objective-C, Objective-C++)
+
+(defcustom lsp-clients-clangd-executable "clangd"
+  "The clangd executable to use.
+Leave as just the executable name to use the default behavior of
+finding the executable with `exec-path'."
+  :group 'lsp-clangd
+  :risky t
+  :type 'file)
+
+(defcustom lsp-clients-clangd-args '()
+  "Extra arguments for the clangd executable."
+  :group 'lsp-clangd
+  :risky t
+  :type '(repeat string))
+
+(defun lsp-clients--clangd-command ()
+  "Generate the language server startup command."
+  (concat lsp-clients-clangd-executable (concat lsp-clients-clangd-args)))
```

Comment:
concat works for strings, here you need `(,lsp-clients-langd-executable ,@lsp-clients-clangd-args)